webapi/Models/Storage/ChatMessage.cs

```diff
@@ -259,4 +259,12 @@ private static bool IsPlan(string response)
         var planPrefix = "proposedPlan\":";
         return response.IndexOf(planPrefix, StringComparison.Ordinal) != -1;
     }
+
+    /// <summary>
+    /// Convert a chat message to a generic string.
+    /// </summary>
+    /// <param name="role"></param>
+    /// <param name="content"></param>
+    /// <returns></returns>
```

Comment:
missing xml doc string for return and params

Reply:
Done in `PromptUtils.FormatChatHistoryMessage`